hw/ip/otbn/rtl/otbn_model.cc

```diff
@@ -0,0 +1,124 @@
+#include "svdpi.h"
```

Comment:
- Can you add a license header?

- We probably should move this file to `hw/ip/otbn/rtl/dv/dpi`